### PR TITLE
contenttypes.yaml includes correct examples with current field types

### LINKED ISSUE #1036: contenttypes.yaml examples are out of date
The `contenttypes.yaml` commented out information is out of date.

### PR DESCRIPTION
**Pull request by I-Valchev**
Fixes #1036

### CODE REVIEW

File: config/bolt/contenttypes.yaml [hunk 1]
```diff
@@ -373,20 +373,21 @@ tests:
...
 # textarea - varchar(32768) - Simple, plain < textarea > field.
-# markdown - varchar(32768) - Simple, plain < textarea > field, which is parsed as Markdown text.
+# markdown - varchar(32768) - wysiwyg element, which is parsed as Markdown text.
 # datetime - datetime - date and time selector widget.
 # date - datetime - date selector widget.
 # select - varchar(256) - select with predefined values
 # templateselect - varchar(256) - select with template filenames
 # checkbox - integer - checkbox-field which is 1 (checked) or 0 (unchecked)
+# set - text (65kb) - Input type for sests. Group fields semantically into a combined set field.
+# Collection - text (65kb) - Input type for collections. Add multiple fields of different types. Useful for lists.
```
Comment (bobdenotter) [inline config/bolt/contenttypes.yaml]: Call me a nitpick, but let's keep it consistently lowercase

```suggestion
# collection - text (65kb) - Input type for collections. Add multiple fields of different types. Useful for lists.
```

File: config/bolt/contenttypes.yaml [hunk 2]
```diff
@@ -418,6 +419,7 @@ tests:
 #
 #contenttype_abc:
 #    name: ContentType Abc
+#    slug: contenttypeabc
```
Comment (bobdenotter) [inline config/bolt/contenttypes.yaml]: I believe we're enforcing now that the `key` should be the same as the `slug`. So, i think the example should say: 

```yaml
#contenttypeabc:
#    name: ContentType Abc
#    slug: contenttypeabc
```

So, both _without_ the `_`.

File: config/bolt/contenttypes.yaml [hunk 3]
```diff
@@ -433,6 +435,7 @@ tests:
 #
 #contenttype_xyz:
 #    name: ContentType Xyz
+#    slug: contenttypexyz
```
Comment (bobdenotter) [inline config/bolt/contenttypes.yaml]: same as above.

File: config/bolt/contenttypes.yaml [hunk 4]
```diff
@@ -373,20 +373,21 @@ tests:
...
+# imagelist - text (65kb) - Input type for imagelists. Add multiple images. Useful for image sliders, galleries, etcetera.
 # image - varchar(256) - image select/upload widget, stored as filename.
 # file - varchar(256) - file select/upload widget, stored as filename.
 # filelist - text (65kb) - Input type for a list of files. Including a description.
 # video - varchar(256) - Video embed widget, stored as filename.
 # html - text (65kb) - wysiwyg element.
 # textarea - varchar(32768) - Simple, plain < textarea > field.
-# markdown - varchar(32768) - Simple, plain < textarea > field, which is parsed as Markdown text.
+# markdown - varchar(32768) - wysiwyg element, which is parsed as Markdown text.
 # datetime - datetime - date and time selector widget.
```
Comment (bobdenotter) [inline config/bolt/contenttypes.yaml]: We removed `datetime` as well, and added `mode: datetime` for adding a time

**Comment by I-Valchev** [comment]
Yes, good points thanks. This should be ok now?

**Pull request merged**